usermods/Tubes/node.h

```diff
@@ -340,30 +440,48 @@ class LightNode {
 #else
         reset();
 #endif
-        statusTimer.stop();
-        quickEspNow.onDataRcvd(onDataReceived);
 
-        Serial.println("Mesh: ok");
+        // Initialising the UI will init the display too.
+#ifdef USERMOD_WLED_TUBES_OLED
```

Comment:
it could.  And it would probably be worth wrapping `Serial.printX` with something that also outputs to the OLED for debug purposes.  Perhaps a separate file / class for sending debug output to either/both.  I used this initially and then was able to get what I needed out of the other serial lines

Reply:
yep, I don't mind wrapping our debug messages to support the either/both output - hopefully it's easy enough to do with a crazy overloaded message like Serial.print/ln

Maybe we should do that in a different diff tho & figure out how to do it right